src/ResourceManager/Network/Commands.Network/Models/PSNetworkInterface.cs

```diff
@@ -39,6 +39,9 @@ public class PSNetworkInterface : PSTopLevelResource, INetworkInterfaceReference
         [Ps1Xml(Target = ViewControl.Table)]
         public bool? EnableIPForwarding { get; set; }
 
+        [Ps1Xml(Target = ViewControl.Table)]
+        public List<string> HostedWorkloads { get; set; }
```

Comment:
Do we need to regenerate the help file because of this change?

Also, where is this property assigned to?

Reply:
Shouldn't we add a test that checks that the property is actually returned by the server?